crates/iota-grpc-api/tests/checkpoint_e2e.rs

```diff
@@ -126,15 +126,17 @@ async fn test_get_epoch_first_checkpoint_sequence_number() {
         match res {
             Ok(cp) => match GrpcNodeClient::deserialize_checkpoint(&cp) {
                 Ok(iota_grpc_api::client::CheckpointContent::Summary(summary)) => {
-                    let epoch = summary.data().epoch;
-                    println!(
-                        "Checkpoint sequence_number: {}, epoch: {}",
-                        cp.sequence_number, epoch
-                    );
-                    all_indices.push(cp.sequence_number);
-                    all_epochs.push(epoch);
-                    if cp.sequence_number > 50 {
-                        break;
+                    if let Some(v1_summary) = summary.as_v1() {
```

Comment:
Should we rather use a `match` here, so we don't miss these places when we add new types?

Reply:
Thanks for pointing this out! Fixed it at [0469376](https://github.com/iotaledger/iota/pull/7511/commits/0469376723d56074ca9fa703d333e4a9cd104fc8).